src/components/MdDatepicker/MdDatepicker.vue

```diff
@@ -127,9 +127,6 @@
   }
 
   .md-datepicker {
-    label {
-      top: 0 !important;
-    }
```

Comment:
We need a class to separate native and vue material datepicker dialog:

![image](https://user-images.githubusercontent.com/29639463/35132934-a52ce572-fd08-11e7-8b0f-17fe34129a08.png)